lib/views/branch-list-view.coffee

```diff
@@ -9,6 +9,7 @@ class ListView extends SelectListView
 
   initialize: (@repo, @data) ->
     super
+    @addClass('git-branch')
```

Comment:
Is there styling that uses this class name? This name is being used by atom already so we should be careful of clashing of styles